src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs

```diff
@@ -2785,6 +2842,43 @@ private void SetColor(Color value)
                 throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        private void SetColorRed(float? value)
+        {
+            if (value == null)
+            {
+                return;
+            }
+
+            Tizen.NUI.Object.SetProperty((System.Runtime.InteropServices.HandleRef)SwigCPtr, View.Property.ColorRed, new Tizen.NUI.PropertyValue((float)value));
+            if (NDalicPINVOKE.SWIGPendingException.Pending)
```

Comment:
could you please check warning messages?

Reply:
We are plan to remove useless `new PropertyValue` and will use `Object.SetProperty(~~, float)` soon. This warning message will be removed when that project finished.